packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -639,6 +639,9 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
       checked: item.checked,
       icon: item.icon,
       iconProps: item.iconProps,
+      onMouseEnter: this._onItemMouseEnter.bind(this, item, true /* isSplitButtonPrimary */),
```

Comment:
It seems like we should be treating the two buttons as more separate than they are. SplitButtonPrimary doesn't have submenu items, and shouldn't be treated as such. Instead of adding this, instead do `{...item, subMenuProps: null, items: null}` in render split button. It keeps it cleaner. You could also parse this further since the icon part of the split button really shouldn't get passed the onclick either.